.lintstagedrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "*.{md,html,css,json}": "prettier --write",
```

Comment:
Hm, hvorfor disse type filene? Burde vi ikke ha prettier write på ts og tsx?

Reply:
Jo, mener at det blir fikset gjennom eslint med prettier plugin

Reply:
La til noen ekstra regler for å fange opp conflicting regler med prettier i eslint nå.

Testet det hos meg ved å disable prettier og eslint i IDEA'en og commit'et en uformatert fil, ser ut som det funker.

Du kan teste branchen på din hvis du gidder, just in case.

Reply:
cool, ser ut som det funker uten at jeg skjønner hvorfor :P